### ISSUE TITLE
bosh-cli is version 3, but gives the option to create confusing bosh2 command

### ISSUE THREAD

**Issue by rheaton** [cloudfoundry/homebrew-tap#46]

**Comment by dpb587-pivotal** [comment]
Yeah, this is a little weird. The option currently exists to provide easier compatibility for those still running older versions of ruby `bosh`.

I was under the impression that those options only show up if a user intentionally asks for more details via `brew options bosh-cli`, is that correct? If so, I think we'd still like to keep the option around for compatibility. But if `--with-bosh2` is showing up frequently and confusing people, we should probably consider an alternative.
